src/main/java/org/terasology/launcher/gui/javafx/ApplicationController.java

```diff
@@ -338,19 +337,6 @@ private void onSync() {
         jobBox.getSelectionModel().select(0);
     }
 
-    /**
-     * Workaround to reset the scroll bar of the given ComboBox to either the selected index or the first element.
-     * <p>
-     * Code taken from: https://stackoverflow.com/a/57885977
-     *
-     * @param cb The {@link ComboBox} to reset the scroll bar for.
-     */
-    private void resetScrollBar(final ComboBox cb) {
```

Comment:
I guess this is just no longer working, right?... No need to find yet another work around.

With the switch to Java 11 we may use more controls (such as https://github.com/controlsfx/controlsfx/wiki/ControlsFX-Features), and we've been thinking about a UI re-design for some time now, anyways.